### PR TITLE
Icon for linux, windows

### PR DESCRIPTION
**Pull request by jdetle**
Addresses #559 somewhat. 
@captainsafia or @rgbkrk should check this out and make sure it doesn't break mac stuff.

### CODE REVIEW

**Comment by jdetle** [comment]
Totally misread the problem. Path to static is different if built then if ran using npm run start. I'll think about how to address both use cases.

File: src/main/launch.js [hunk 1]
```diff
@@ -10,12 +10,20 @@ export function deferURL(event, url) {
   event.preventDefault();
   shell.openExternal(url);
 }
+let iconName = '';
+
+if (process.platform == 'win32') {
+  iconName = './static/icon.ico';
+} else if (process.platform == 'linux') {
+  iconName = './static/icon.png';
```
Comment (rgbkrk) [inline src/main/launch.js]: This should be ../

Comment (jdetle) [inline src/main/launch.js]: For whatever reason that did not actually work. Is `static/icon.png` better?

**Comment by jdetle** [comment]
@captainsafia can you check this out and make sure it doesn't break the icons for mac machines?

**Comment by rgbkrk** [comment]
I'm going to assume yes, which means we'll find out soon.

**Pull request merged**